### DISCUSSION TITLE
Weather dashboard

### DISCUSSION THREAD

**Discussion by jsenecal** [Madelena/hass-config-public#20] [Q&A] [answered]
Hi @Madelena , Just saw your demo on [Home Assistant 2023.7 Release Party](https://www.youtube.com/watch?v=r0DLyU-4nWo&pp=ygUjaG9tZSBhc3Npc3RhbnQgMjAyMy43IHJlbGVhc2UgcGFydHk%3D);

I've been a user of your inspired config and theme for quite some time now and noticed that you added quite a lot to your home config :)

Is there any plan to share your weather view to the world?

**Comment by jsenecal** [comment, marked as answer]
Its there now! Thanks @Madelena 

https://github.com/Madelena/hass-config-public/blob/main/ui/views/weather.yaml

**Comment by Madelena** [comment]
@jsenecal You're welcome! Sorry it took me a while 😬
